pkg/api/customization/project/project_actions.go

```diff
@@ -59,6 +63,16 @@ func (h *Handler) Actions(actionName string, action *types.Action, apiContext *t
 		return h.setPodSecurityPolicyTemplate(actionName, action, apiContext)
 	case "exportYaml":
 		return h.ExportYamlHandler(actionName, action, apiContext)
+	case "viewMonitoring":
+		if !canUpdateProject() {
```

Comment:
Is there any reason to check the updateProject privilege here?

Reply:
I want to ensure accessor has the same privilege as `disableMonitoring` and `editMonitoring`.